components/MessageContextMenuWrappers/MessageContextMenuWrapper.tsx

```diff
@@ -82,14 +96,31 @@ export const MessageContextMenuWrapper: FC<MessageContextMenuWrapperProps> = ({
     });
     if (!isAttachment && !isTransaction) {
       items.push({
-        title: "Copy message",
+        title: "Copy",
         systemIcon: "doc.on.doc",
         onPress: handleCopyMessage,
       });
     }
+    if (frameURL) {
+      items.push({
+        title: "Share",
```

Comment:
Don't need to change, for this PR, but can we start putting new features into the translations: i18n/translations/en.ts
and using the translate function

Reply:
For sure. I discovered that function the other night and went to town... https://github.com/ephemeraHQ/converse-app/pull/660